EMIP/1001-2000/EMIP01971.xml

```diff
@@ -57,6 +74,64 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         </extent>
                      </supportDesc>
                   </objectDesc>
+                  <handDesc>
+                  <handNote xml:id="h1" script="Ethiopic">
+                  <seg type="rubrication">
+                  <foreign xml:lang="gez">ኢግዚአብሔር፡</foreign>
+                  <desc>The word God is not rubricated in the first three works</desc>
+                  <foreign xml:lang="gez">ማርያም፡</foreign>
+                  </seg>
+                  <desc>The word Mary is rubricated in the final two works</desc>
+                  <desc>Columetric layout of text: <locus target="#138rv"/> (tenth Biblical Canticle)</desc>
+                  </handNote>
+                  </handDesc>
+                  <decoDesc>
+                  <decoNote type="frame" xml:id="d01">
+                  Decorative designs:
```

Comment:
These are elaborate ornamental bands, nicely done (a distinctive feature of a special group of Psalters); I would suggest to write them out, specifying a little bit (motifs: floral, zoomorphic, anthropomorphic).